workflow/envs/kendall.yml

```diff
@@ -0,0 +1,12 @@
+name: kendall
```

Comment:
Let's combine the dependencies you need for kendall into the `sc_e2g.yml` so that the repo only needs 1 conda environment

Reply:
Great idea!
Appending the following lines to the sc_e2g.yml should work 
```
  - bioconductor-genomation
  - r-foreach
```